tests/devices/test_default_qubit_autograd.py

```diff
@@ -280,6 +280,22 @@ def circuit(a, b):
         )
         assert np.allclose(res, expected_grad, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize("x, shift", [(0.0, 0.0), (0.5, -0.5)])
+    def test_hessian_at_zero(self, x, shift):
+        """Tests that the Hessian at vanishing state vector amplitudes
+        is correct."""
+        dev = qml.device("default.qubit.autograd", wires=1)
+
+        @qml.qnode(dev, interface="autograd")

```

Comment:
```suggestion
        @qml.qnode(dev, interface="autograd", diff_method="backprop")
```
(to protect this test in case defaults change)